### PR TITLE
xray-toolset: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Close https://github.com/BlackArch/blackarch/issues/3310

### CODE REVIEW

File: packages/xray-toolset/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+pkgdesc='A complete security assessment tool that supports common web security issues scanning and custom POC.'
+arch=('x86_64')
+groups=('blackarch' 'blackarch-scanner' 'blackarch-webapp')
+url='https://github.com/chaitin/xray'
+license=('custom:unknown')
+depends=()
+makedepends=('git')
+source=("git+https://github.com/chaitin/xray.git"
+        "git+https://github.com/chaitin/xray-plugins.git"
+        "https://github.com/chaitin/xray/releases/download/xpoc-0.1.0/xpoc_linux_amd64"
```
Comment (noraj) [inline packages/xray-toolset/PKGBUILD:16]: add xpoc as dependency and built it https://github.com/chaitin/xpoc

File: packages/xray-toolset/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "https://github.com/chaitin/xray/releases/download/xapp-0.0.2/xapp_linux_amd64"
```
Comment (noraj) [inline packages/xray-toolset/PKGBUILD:17]: add xapp as dependency and build it https://github.com/chaitin/xapp

File: packages/xray-toolset/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        "https://github.com/chaitin/xray-plugins/releases/download/xlint-0.0.3/xlint_linux_amd64")
```
Comment (noraj) [inline packages/xray-toolset/PKGBUILD:18]: add xray-plugins as dependency and built it https://github.com/chaitin/xray-plugins/

**Review by noraj — CHANGES_REQUESTED** [review]
create the packages for the dependencies instead of using pre-build binaries

File: packages/xray-toolset/PKGBUILD (hunk 1 above)
Comment (D3vil0p3r) [reply packages/xray-toolset/PKGBUILD:16]: The `xpoc` repository has not source code to build the binary. Should I create a dedicated package for each one of them and still use the pre-build binaries?

Comment (noraj) [reply packages/xray-toolset/PKGBUILD:16]: True. Then if it's closed-source, we would avoid shipping pre-built binaries from untrusted sources.

**Comment by noraj** [comment]
Closed source software (see https://github.com/chaitin/xray/blob/master/README_EN.md):

> Note: xray is not open source, just download the built binary files directly. The repository mainly contains community-contributed POCs. Each xray release will automatically package them.

File: packages/xray-toolset/PKGBUILD (hunk 1 above)
Comment (D3vil0p3r) [reply packages/xray-toolset/PKGBUILD:16]: I agree. At this point I would close this PR. Can you please close https://github.com/BlackArch/blackarch/issues/3310 by explicitly writing the reason you just mentioned?

**Pull request closed without merging**